.teamcity/_self/projects/WebApp.kt

```diff
@@ -614,6 +614,27 @@ object Translate : BuildType({
 			"""
 			dockerImage = "%docker_image_e2e%"
 		}
+		bashNodeScript {
+			name = "Notify GlotPress Translate build is ready"
+			scriptContent = """
+				if [[ "%teamcity.build.branch.is_default%" == "true" ]]; then
+					exit 0
+				fi
```

Comment:
Just to check — this is because every PR will already have run the rest of this step, and the POT artifacts would already have been passed to GlotPress, making it redundant to do it for `trunk` again? 🙂

Reply:
The `localci/-relay-new-strings-to-gh` endpoint updates the status of the translations and posts to the PR, therefore it's not relevant to trunk TeamCity builds.
We already check if it's the default branch in `gp-localci` plugin, but I thought better to also prevent the request in the build script.